modules/claims_api/spec/requests/v2/veterans/disability_compensation_request_spec.rb

```diff
@@ -27,11 +27,14 @@
   describe '#526' do
     context 'submit' do
       let(:claim_date) { (Time.zone.today - 1.day).to_s }
+      let(:anticipated_separation_date) { 2.days.from_now.strftime('%Y-%m-%d') }
       let(:data) do
         temp = Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans', 'disability_compensation',
                                'form_526_json_api.json').read
         temp = JSON.parse(temp)
         temp['data']['attributes']['claimDate'] = claim_date
+        temp['data']['attributes']['serviceInformation']['reservesNationalGuardService']['title10Activation']['anticipatedSeparationDate'] = # rubocop:disable Layout/LineLength
```

Comment:
After adding validation to make sure that date is _always_ in the future if present I saw a bunch of tests failing. There is some testing for when it is _not_ in the future which does alter the data object, but I figured that would be the cleanest way to make sure it was always in the future when running the test suite.  Otherwise I have to set it in the file for a year we will never reach, this just felt like a cleaner solution.

Reply:
where do we stand on this? just curious if we're happy with it. I believe this was approved previously @stiehlrod , but looks like the many, many merge conflict resolutions have long dismissed it.